symphony-bdk-examples/bdk-core-examples/src/main/resources/rsa-private-fxBot.pem

```diff
@@ -0,0 +1,46 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Rename this file to `sample-privatekey.pem`

Reply:
Oops! This file was commit by a mistake.